test-network/README.md

```diff
@@ -0,0 +1,50 @@
+## Running the test network
+
+Use the `./network.sh` script to stand up a simple Fabric test network. The
+network has two peer peer organizations with one peer each and a single node
+raft ordering service. You can also use the script to create channels, and deploy
+the fabcar chaincode on those channels. The test network is being introduced in
```

Comment:
Per @lehors suggestion, added a sentence that this will be a long term replacement for BYFN.